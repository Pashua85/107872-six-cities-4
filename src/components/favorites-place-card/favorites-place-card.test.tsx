import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import FavoritesPlaceCard from './favorites-place-card';
import places from '../../mock/test-offers';

Enzyme.configure({
  adapter: new Adapter()
});

describe(`FavoritesPlaceCard`, () => {
  it(`should render correctly`, () => {
    const wrapper = shallow(
        <FavoritesPlaceCard
          key={2}
          place={places[0]}
          onCardHover={() => {}}
          onCardUnhover={() => {}}
        />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
