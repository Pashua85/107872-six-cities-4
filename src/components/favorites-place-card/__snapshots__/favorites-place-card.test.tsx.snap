// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavoritesPlaceCard should render correctly 1`] = `ShallowWrapper {}`;
